dir.props

```diff
@@ -27,8 +27,8 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <RoslynVersion>1.0.0-rc3-20150510-01</RoslynVersion>
-    <RoslynPackageName>Microsoft.Net.ToolsetCompilers</RoslynPackageName>
+    <RoslynVersion>2.0.0-beta3</RoslynVersion>
```

Comment:
Can we get all these changes out of repo specific dir.props files and instead include them in the common BuildTools props files? That will make updating compilers much simpler for all.